### ISSUE TITLE
The app is not shown in recent apps when minimized

### ISSUE THREAD

**Issue by martofeld** [BranchMetrics/sesame_issues#58]
I was going back and forth between the settings and the home to test my new configurations and every time I tried to go back to the settings I had to go through the app instead of the recents apps switcher

**Comment by philwall2** [comment]
Yeah, it's annoying. FWIW it's a tradeoff situation. The problem is if we register Sesame to show in Recents then it will pretty much always be there. So we don't. But that means if you're using Settings frequently it's not very accessible. We decided hiding it would be less annoying.

The other option would be to separate out the Settings and Sesame Search/Shortcut activities, but that's a ton of work/complexity.

I'll bring it up w/ Steve again.

**Comment by martofeld** [comment]
@philwall2 In my experience it was the first time using Sesame and I was trying to understand how each change affected the search, and it was tedious to have to go back and forth into the settings. 

What do you mean by "separating Settings and Search"? Are they just a big activity?

**Comment by philwall2** [comment]
Yes, we have a single point of entry for Sesame that lets us do all kinda of things quickly. TBH there's probably a way to hack it so only Settings would show in recents and not Search. We'll take a look.

**Comment by martofeld** [comment]
I'd love to help develop this awesome app. Do you think you could take an extra enthusiastic Android Developer?

**Comment by philwall2** [comment]
Hey @martofeld we've tried that a few times and despite people's best intentions they seem to flake after we've invested a couple weeks bringing them up to speed, or they want $$.
